packages/catalog-search/package.json

```diff
@@ -10,7 +10,17 @@
     "last 2 versions",
     "ie 11"
   ],
+  "watch": {
+    "build": {
+      "patterns": [
+        "src"
+      ],
+      "extensions": "js,jsx"
+    }
+  },
   "scripts": {
+    "dev": "npm-watch build",
```

Comment:
This NPM script begins watching for any changes in the `src` files matching extensions `.js` and `.jsx` and executes `npm run build` on any of these changes. This is used alongside with Lerna to watch all package source files and re-transpile as needed to ensure `dist` directories always exist in symlinking.